src/Compute/Compute/help/New-AzVM.md

```diff
@@ -440,6 +440,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ProximityPlacementGroup
+{{ Fill ProximityPlacementGroup Description }}
```

Comment:
Need to fill in the description here, like The name or resource Id of the Proximity Placement Group to use with this VM.